platform/tyan-s1706/pkg/gpio/platform.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gpio
+
+import (
+	"github.com/u-root/u-bmc/pkg/aspeed"
+)
+
+var (
+	linePortMap = map[string]uint32{
+		"UNKN_BOOT1":          aspeed.GpioPort("A2"),
+		"UNKN_PWR_CAP":        aspeed.GpioPort("A3"),
+		"Faspeed":             aspeed.GpioPort("B3"),
```

Comment:
Hah, this used to be "FAST_PROCHOT" and got replaced :-). All these should go and be replaced with lines from the datasheets.